projects/ziti-console-lib/src/lib/features/wrappers/zac-wrapper.service.ts

```diff
@@ -812,6 +813,26 @@ export class ZacWrapperService extends ZacWrapperServiceClass {
         });
     }
 
+    redefineObject(obj) {
```

Comment:
`redefineObject` seems like a strange name to me. it feels more like a `resetObject` to me, but maybe that's my weak js-foo?